<ui version="4.0" >
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>searchForEmp</class>
 <widget class="QWidget" name="searchForEmp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search for Employees</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_3" >
     <property name="margin" >
      <number>12</number>
     </property>
     <property name="spacing" >
      <number>12</number>
     </property>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" colspan="2" >
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="_searchLit" >
             <property name="text" >
              <string>S&amp;earch for:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy" >
              <cstring>_search</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="XLineEdit" name="_search" />
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="XCheckBox" name="_showInactive" >
           <property name="text" >
            <string>Show &amp;Inactive Employees</string>
           </property>
          </widget>
         </item>
         <item rowspan="2" row="1" column="1" >
          <widget class="QGroupBox" name="_searchGroup" >
           <property name="title" >
            <string>Search through</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="XCheckBox" name="_searchCode" >
              <property name="text" >
               <string>Employee Codes</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="XCheckBox" name="_searchMgr" >
              <property name="text" >
               <string>Manager Codes and Numbers</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="XCheckBox" name="_searchNumber" >
              <property name="text" >
               <string>Employee Numbers</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="XCheckBox" name="_searchDept" >
              <property name="text" >
               <string>Departments</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="XCheckBox" name="_searchName" >
              <property name="text" >
               <string>Employee Name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="XCheckBox" name="_searchShift" >
              <property name="text" >
               <string>Shifts</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="WarehouseGroup" name="_warehouse" >
           <property name="fixedSize" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_close" >
           <property name="text" >
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_print" >
           <property name="text" >
            <string>Print</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QFrame" name="_frame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_empLit" >
            <property name="text" >
             <string>Employees:</string>
            </property>
            <property name="buddy" >
             <cstring>_emp</cstring>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>7</number>
            </property>
            <item>
             <widget class="XTreeWidget" name="_emp" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="_new" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>&amp;New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_edit" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>&amp;Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_view" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>&amp;View</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <customwidgets>
  <customwidget>
   <class>WarehouseGroup</class>
   <extends>QGroupBox</extends>
   <header>warehousegroup.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_search</tabstop>
  <tabstop>_searchCode</tabstop>
  <tabstop>_searchNumber</tabstop>
  <tabstop>_searchMgr</tabstop>
  <tabstop>_searchDept</tabstop>
  <tabstop>_searchShift</tabstop>
  <tabstop>_showInactive</tabstop>
  <tabstop>_emp</tabstop>
  <tabstop>_new</tabstop>
  <tabstop>_edit</tabstop>
  <tabstop>_view</tabstop>
  <tabstop>_print</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_emp</sender>
   <signal>valid(bool)</signal>
   <receiver>_view</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>searchForEmp</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>653</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>348</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
